locales/ro_RO/LC_MESSAGES/duckduckgo.po

```diff
@@ -97,19 +99,21 @@ msgstr "%sClick dreapta%s în bara de căutare"
 
 msgctxt "install-extension"
 msgid "1. Open %sDownloads%s"
-msgstr "1. Deschideți %sDescărcări%s"
+msgstr "1. Deschide %sDescărcări%s"
 
 msgctxt "install-extension"
 msgid "2. Double-click %sduckduckgo.safariextz%s"
 msgstr "Dublu click pe %sduckduckgo.safariextz%s"
 
 #. https://duckduckgo.com/params in the end under " Source:"
+#, fuzzy
 msgid "A string to identify the source."
 msgstr "Un șir pentru a identifica sursa."
 
+#, fuzzy
 msgctxt "settings"
 msgid "ATB Dismiss"
```

Comment:
What's an ATB?

Reply:
@alexei I think it's about the anonymous A/B test they have been doing as a way to improve DuckDuckGo (reference: https://duck.co/help/privacy/atb)

Reply:
@epsimatt yeah, I looked it up on the Internet and found there was an `atb` query param used for A/B tests. I wonder now what does it mean by `ATB Dismiss` 'cause I'm not sure how to translate that

Reply:
This one needs a little background.

These tokens are extracted from data which describe user settings. Usually these strings are presented directly on settings pages. I think this "setting" is to note that you have dismissed the prompt to install the DDG browser extension, but it doesn't make sense to make it a configurable setting to re-enable this prompt.

So, the setting needs a string to describe it, but a couple of those strings do not make it to presentation. A note is required for this on the pot file.

In short, I think you are free to ignore this one.